BrainPortal/config/oidc.yml.erb

```diff
@@ -0,0 +1,19 @@
+Keycloak:
+  # A OpenID provider can be enabled
+  enabled:                            true
+  # Configuration of the OpenID provider
+  authorize_uri:                      "http://localhost:8080/realms/.../protocol/openid-connect/auth"
+  token_uri:                          "keycloak:8080/realms/.../protocol/openid-connect/token"
+  logout_uri:                         "http://localhost:8080/realms/.../protocol/openid-connect/logout"
+  # Scope used when called the OpenID provider when autheticate
+  scope:                              "openid email profile"
+  # OpenID provider id and secret need to be passed via environment variables
+  client_id:                          <%= ENV["keycloak_client_id"] %>
```

Comment:
The examples of ruby code to embed environment variables in the OIDC YAML file should also zap the environment variables, so that they don't get propagated inside the app.

Reply:
e.g.

```
client_id: <%= ENV.delete 'keycloak_client_id' %>
```

Reply:
When I booted my server, I got this in the initial boot messages:

```
C> Loading OIDC configuration...
C>      - ERROR: Cannot load OIDC configuration.
C>        Missing keys client_secret, client_id in OIDC config: Keycloak
C> CBRAIN BrainPortal validation completed, 2024-07-16 11:34:22 -0400
```

Yet the application continue to boot. That's wrong, if the admin does not configure this important confi file correctly, the server shoudl not continue!